index.html

```diff
@@ -8,26 +8,176 @@
+              <span class="work-name">CANOPY</span>
+              <ul class="role">
+                <li>Back End Dev</li>
+                <li>2015</li>
+              </ul>
+            </div>
+            <p>
+              A daily selection of privately personalized reads; no accounts or
+              sign-ups required.
+            </p>
+            <ul class="works-technologies">
+              <li>html</li>
+              <li>css</li>
+              <li>javascript</li>
+            </ul>
+            <a href="#" class="btn">See Project</a>
+          </div>
+        </div>
+      </div>
+
+      <div class="container works">
+        <div class="work-item">
+          <img src="./img/Snapshoot Portfolio3.svg" alt="project 2 image" />
+          <div class="work-item-details">
+            <h2>Tonic</h2>
+            <div class="work-item-intro">
+              <span class="work-name">CANOPY</span>
+              <ul class="role">
+                <li>Back End Dev</li>
+                <li>2015</li>
+              </ul>
+            </div>
+            <p>
+              A daily selection of privately personalized reads; no accounts or
+              sign-ups required.
+            </p>
+            <ul class="works-technologies">
+              <li>html</li>
+              <li>css</li>
+              <li>javascript</li>
+            </ul>
+            <a href="#" class="btn">See Project</a>
+          </div>
+        </div>
+      </div>
+
+      <div class="container works">
+        <div class="work-item">
+          <img src="./img/Snapshoot Portfolio4.svg" alt="project 2 image" />
+          <div class="work-item-details">
+            <h2>Tonic</h2>
+            <div class="work-item-intro">
+              <span class="work-name">CANOPY</span>
+              <ul class="role">
+                <li>Back End Dev</li>
+                <li>2015</li>
+              </ul>
+            </div>
+            <p>
+              A daily selection of privately personalized reads; no accounts or
+              sign-ups required.
+            </p>
+            <ul class="works-technologies">
+              <li>html</li>
+              <li>css</li>
+              <li>javascript</li>
+            </ul>
+            <a href="#" class="btn">See Project</a>
+          </div>
+        </div>
+      </div>
+    </section>
+
+    <section class="about-section">
+      <div class="container about">
+        <h2>About<br />Myself</h2>
         <p>
           I’m a software developer! I can help you build a product , feature or
           website Look through some of my work and experience! If you like what
           you see and have a project you need coded, don’t hestiate to contact
           me.
         </p>
         <a href="#" class="connect">Lets Connect</a>
-        <ul class="headline-socials">
+        <ul class="about-socials">
           <li>
             <a href="#"><img src="./img/Vector.svg" alt="twitter logo" /></a>
           </li>
```

Comment:
Please consider adding active links to all of your social media pages listed in this list, an active link will make people e.g recruiters find it easy to connect with you on platforms such as [GitHub](https://github.com) or [LinkedIn](https://linkedin.com)

Reply:
pls, sir are saying that I should wrap my list items inside an anchor tag?? Because they already have active links inside of them. I just want to be clear.

Reply:
@Linushaddai99 I'm suggesting that you make the `a` tags active links that lead to your social media pages. Please note that currently, they are not functional links that can lead someone to your social media pages.